### ISSUE TITLE
Web component: Masthead Customizable IBM Logo - On keyboard navigation, tooltip is not appearing. 

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4800]
## Detailed description

Actual Result:
When focus reaches to IBM logo on Safari, tooltip is not appearing.

Expected Result:
Tooltip should be accessible for keyboard user

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/bd180994-1b74-4a59-b6c7-4d8eb4560d02)

**Comment by IgnacioBecerra** [comment]
@Praveen-sr Seems like this has been fixed, could you re-test?
![Screen Shot 2021-01-11 at 4.40.36 PM.png](https://images.zenhubusercontent.com/5f63c83b3584b9bd910ec115/53640637-146e-4c90-a9e0-b4ac89040562)

**Comment by Praveen-sr** [comment]
@IgnacioBecerra Re tested this on Safari and Firefox,
On safari - its working 
On Firefox, Windows - its not working

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/d38d6a31-5773-433a-8e88-e429e9bb1f97)
